### PR TITLE
Add language setting and responsive option

### PR DESCRIPTION
**Pull request by samiraguiar**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by rbuckton** [comment]
@samiraguiar Could you provide a link to documentation that provides context for this change?

**Comment by samiraguiar** [comment]
@rbuckton Sure, the docs for both additions are under [AmCharts.AmChart.responsive](https://docs.amcharts.com/3/javascriptcharts/AmChart#responsive) and [AmCharts.AmChart.language](https://docs.amcharts.com/3/javascriptcharts/AmChart#language).

File: amcharts/AmCharts.d.ts [hunk 1]
```diff
@@ -1034,6 +1036,10 @@ If you do not set properties such as dashLength, lineAlpha, lineColor, etc - val
 
         /**   Adds title to the top of the chart. Pie, Radar positions are updated so that they won't overlap. Plot area of Serial/XY chart is also updated unless autoMargins property is set to false. You can add any number of titles - each of them will be placed in a new line. To remove titles, simply clear titles array: chart.titles = []; and call chart.validateNow() method. text - text of a title size - font size color - title color alpha - title opacity bold - boolean value indicating if title should be bold. */
         addTitle(text: string, size: number, color: string, alpha: number, bold: boolean);
+        /** Allows changing language easily.
+         * Note, you should include language js file from amcharts/lang or ammap/lang folder and then use variable name used in this file, like chart.language = "de";
+         * Note, for maps this works differently - you use language only for country names, as there are no other strings in the maps application. */
+        language(lang: string);
```
Comment (vvakame) [inline amcharts/AmCharts.d.ts]: Is this right?
https://docs.amcharts.com/3/javascriptcharts/AmChart#language
It looks like `language: string;`

Comment (samiraguiar) [inline amcharts/AmCharts.d.ts]: Yep, you're right, fixed.

Comment (vvakame) [inline amcharts/AmCharts.d.ts]: well...
I found this text from document.
>  like chart.language = "de";

Is this mean `language: string;`(property)? or not `language(lang: string): void;`(method)?

Comment (samiraguiar) [inline amcharts/AmCharts.d.ts]: Ops, my bad! It's actually a property.

**Pull request merged**